packages/apps/docs/src/components/BottomPageSection/components/EditPage.tsx

```diff
@@ -8,15 +9,21 @@ interface IProps {
 
 export const EditPage: FC<IProps> = ({ editLink }) => {
   if (!editLink) return null;
+  const registerClick = (): void => {
+    analyticsEvent(EVENT_NAMES['click:edit_page']);
+  };
   return (
-    <Button
-      as="a"
-      href={editLink}
-      target="_blank"
-      rel="noreferrer"
-      title="Edit this page"
-    >
-      Edit this page
-    </Button>
+    <span onClick={registerClick}>
+      <Button
+        as="a"
+        id="bla"
+        href={editLink}
+        target="_blank"
+        rel="noreferrer"
+        title="Edit this page"
+      >
+        Edit this page
+      </Button>
+    </span>
```

Comment:
this looks bit weird to me! having `onClick` on span and also having link inside. Instead of having `Button` as `a` element you could keep it as a default `Button` and as part of `onClick` could use [`useRouter`](https://nextjs.org/docs/app/api-reference/functions/use-router) hook to push url changes and can also add analytics event. ideally it should work.